cli/configure/configure.go

```diff
@@ -335,10 +335,12 @@ func configureNonExistentCmd(rootCmd *cobra.Command, cmdCtx *cmdcontext.CmdCtx,
 	}
 
 	helpCmd := util.GetHelpCommand(rootCmd)
-	if module, ok := (*modulesInfo)[externalCmd]; ok {
+	externalCmdPath := rootCmd.Name() + " " + externalCmd
+	if module, ok := (*modulesInfo)[externalCmdPath]; ok {
 		if !module.IsInternal {
-			rootCmd.AddCommand(newExternalCommand(cmdCtx, modulesInfo, externalCmd, nil))
-			helpCmd.AddCommand(newExternalCommand(cmdCtx, modulesInfo, externalCmd,
+			rootCmd.AddCommand(newExternalCommand(cmdCtx, modulesInfo, externalCmd,
```

Comment:
Do we have integration tests to check the functionality of the help from external commands?

Reply:
Sorry for the late response: we have [integration test](https://github.com/tarantool/tt/blob/59404d4085e6657f10f3a0ac542be26c70d9ef97/test/integration/help/test_help.py#L41) for this case.